tests/test_main.py

```diff
@@ -1 +1,45 @@
-# write your code here
+from app.main import get_human_age
+
+
+def test_cat_age_under_15():
+    goals = get_human_age(14, 29)
+
+    assert goals[0] == 0
+
+
+def test_dog_age_under_15():
+    goals = get_human_age(25, 14)
+
+    assert goals[1] == 0
```

Comment:
You can merge theese functions into one, since the corellation of dog and cat years is the same

Reply:
Changes provided!